bukuserver/server.py

```diff
@@ -466,93 +466,6 @@ def search_bookmarks():
     return res
 
 
```

Comment:
Removing the code makes the imports in line 6 and 16 of the file redundant. That's the reason behind build failure.